src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/config/WebClientConfig.kt

```diff
@@ -62,17 +62,6 @@ class WebClientConfig(
     authorizedClientManagerRequestScope(clientRegistrationRepository, authorizedClientRepository)
   )
 
-  @Bean
-  @RequestScope
-  @Profile("!app-scope")
-  fun communityApiClientCreds(
-    clientRegistrationRepository: ClientRegistrationRepository,
-    authorizedClientRepository: OAuth2AuthorizedClientRepository,
-  ): WebClient? = getClientCredsWebClient(
-    "$communityApiUrl/secure",
-    authorizedClientManagerRequestScope(clientRegistrationRepository, authorizedClientRepository)
-  )
```

Comment:
We can't use a request scope bean as this puts the NOMIS user name into the client creds and Delius doesn't recognise them,